spec/models/step_by_step_page_spec.rb

```diff
@@ -20,6 +20,29 @@
     end
   end
 
+  describe "associations" do
+    let(:stub_user) { create(:user) }
+
+    before do
+      step_by_step_page.review_requester_id = stub_user.uid
+    end
+
+    it "should allow return of review requester name" do
+      step_by_step_page.save
+
+      expect(step_by_step_page.review_requester.name).to eq stub_user.name
+    end
+
+    it "should allow return of reviewer name" do
+      reviewer_user = create(:user)
+      step_by_step_page.reviewer_id = reviewer_user.uid
+
+      step_by_step_page.save
+
+      expect(step_by_step_page.review_requester.name).to eq reviewer_user.name
```

Comment:
Shouldn't this be?

```suggestion
      expect(step_by_step_page. reviewer.name).to eq reviewer_user.name
```

Reply:
Yes, it should be.

I didn't catch this because both users had a name of `nil`.